package.json

```diff
@@ -22,32 +22,32 @@
     "trailjs"
   ],
   "dependencies": {
-    "bluebird": "^3.3.1",
-    "bookshelf": "^0.9.1",
-    "bookshelf-schema": "^0.2.1",
-    "joi": "^8.0.5",
-    "knex": "^0.10.0",
-    "lodash": "^4.5.0",
-    "trailpack": "^1.0.0-beta-5",
-    "trailpack-datastore": "^1.0.0",
+    "bluebird": "^3.3.5",
+    "bookshelf": "^0.9.5",
+    "bookshelf-schema": "^0.2.3",
+    "joi": "^8.1.0",
+    "knex": "^0.11.3",
+    "lodash": "^4.12.0",
+    "trailpack": "^1.0.2",
```

Comment:
Normally you don't need trailpack dependency because you extends `trailpack-datastore`. Can you check ?